src/pages/atomicdex/api/v20-dev/lightning/payments/index.mdx

```diff
@@ -0,0 +1,598 @@
+	"error_path": "send_payment.lightning",
+	"error_trace": "send_payment:102] lightning:232]",
+	"error_type": "PaymentError",
+	"error_data": "Error paying invoice: Routing(LightningError { err: 'Cannot route when there are no outbound routes away from us ', action: IgnoreError })",
+	"id": 762
+}
+```
+
+### PaymentError (keysend - no path to destination)
+```
+{
+	"mmrpc": "2.0",
+	"error": "Payment error: Keysend error: Routing(LightningError { err: "Failed to find a path to the given destination", action: IgnoreError })",
+	"error_path": "send_payment.lightning",
+	"error_trace": "send_payment:107] lightning:256]",
+	"error_type": "PaymentError",
+	"error_data": "Keysend error: Routing(LightningError { err: "Failed to find a path to the given destination", action: IgnoreError })",
+	"id": 762
+}
+```
+
+
+### PaymentError (invoice - no path to destination)
+```
+{
+	"mmrpc": "2.0",
+	"error": "Payment error: Error paying invoice: Routing(LightningError { err: 'Failed to find a path to the given destination', action: IgnoreError })",
+	"error_path": "send_payment.lightning",
+	"error_trace": "send_payment:102] lightning:232]",
+	"error_type": "PaymentError",
+	"error_data": "Error paying invoice: Routing(LightningError { err: 'Failed to find a path to the given destination', action: IgnoreError })",
+	"id": 762
+}
+```
+
+
+
+
+### InvalidRequest (request is missing a required field)
+```
+{
+	"mmrpc": "2.0",
+	"error": "Error parsing request: missing field `payment`",
+	"error_path": "dispatcher",
+	"error_trace": "dispatcher:109]",
+	"error_type": "InvalidRequest",
+	"error_data": "missing field `payment`",
+	"id": 762
+}
+```
+
+### InvalidRequest (A required field is invalid)
+```
+{
+	"mmrpc": "2.0",
+	"error": "Error parsing request: ParseError(Bech32Error(MissingSeparator))",
+	"error_path": "dispatcher",
+	"error_trace": "dispatcher:109]",
+	"error_type": "InvalidRequest",
+	"error_data": "ParseError(Bech32Error(MissingSeparator))",
+	"id": 762
+}
+```
+
+</CollapsibleSection>
+
+
+## Get Payment Details {{label : 'lightning::payments::get_payment_details', tag : 'API-v2'}}
+
+The `lightning::payments::get_payment_details` method returns details about a lightning payment from a given `payment_hash`.
+
+
+### Request Parameters
+
+| Parameter            | Type    | Description                                                                                                                           |
+|----------------------|---------|---------------------------------------------------------------------------------------------------------------------------------------|
+| coin                 | string  | Ticker of the coin to query.                                                                                                          |
+| payment_hash         | string  | The hexidecimal string which identifies a invoice. The `payment_hash` is returned from a `lightning::payments::send_payment` request. |
```

Comment:
The `payment_hash` is returned from a `lightning::payments::send_payment` request if the payment was sent by us or from a `lightning::payments::generate_invoice` request if the payment is received or will be received by us.